python_modules/libraries/dagster-msteams/dagster_msteams_tests/test_resources.py

```diff
@@ -2,26 +2,26 @@
 
 from dagster import op
 from dagster._utils.test import wrap_op_in_graph_and_execute
-from dagster_msteams import msteams_resource
+from dagster_msteams import MSTeamsResource, msteams_resource
 from mock import patch
 
 
 @patch("dagster_msteams.client.TeamsClient.post_message")
 def test_msteams_resource(mock_teams_post_message, json_message, teams_client):
     @op(required_resource_keys={"msteams"})
-    def msteams_solid(context):
+    def msteams_op(context):
         assert context.resources.msteams
         body = {"ok": True}
         mock_teams_post_message.return_value = {
             "status": 200,
             "body": json.dumps(body),
             "headers": "",
         }
-        teams_client.post_message(json_message)
+        context.resources.msteams.post_message(json_message)
```

Comment:
It comes from this [pytest fixture](https://github.com/dagster-io/dagster/blob/42b56c7059db3ed002a1a2129710f229dcf2b08f/python_modules/libraries/dagster-msteams/dagster_msteams_tests/conftest.py#L23). I don't think the resource was actually directly under test before? The behavior here confuses me since it seems like it was intended to test the passed `context.resources.msteams` from resources but instead just directly calls this fixture client.